code/modules/clothing/masks/gasmask.dm

```diff
@@ -45,15 +54,73 @@ GLOBAL_LIST_INIT(clown_mask_options, list(
 	QDEL_LAZYLIST(gas_filters)
 	return..()
 
+/obj/item/clothing/mask/gas/equipped(mob/equipee, slot)
+	if(cig)
+		cig.equipped(equipee, slot)
+	return ..()
+
+/obj/item/clothing/mask/gas/adjustmask(mob/living/carbon/user)
+	if(cig)
+		user.visible_message(
+				span_warning("You need to remove [cig] before you can adjust the mask")
+		)
+		return
+	return ..()
```

Comment:
```suggestion
	if(isnull(cig))
		return ..()
	user.visible_message(span_warning("You need to remove [cig] before you can adjust the mask!"))
```

This early return is a lot cleaner, as well as takes up less space. fixed up the formatting as well. could do a `balloon_alert()` instead too.

Reply:
Gonna leave it as a span warning because that's what the code that was already there was using, if that makes sense?

Reply:
Aaaand literally just saw your other comment now. I'll just switch them all to balloon alerts then.